rpc/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+mod blob;
+pub mod client;
+mod error;
+mod header;
+mod share;
+
+pub use error::{Error, Result};
+
+pub mod prelude {
```

Comment:
usually prelude are re-exports for convenience. if we are going to have it i believe that modules should be public or exported on the base.

Reply:
reexported from the base